README.md

```diff
@@ -11,3 +11,10 @@ page](https://docs.github.com/en/get-started/writing-on-github/working-with-adva
 might help with the notation for mathematical expressions.
 
 $T(n) \in O(f(n)) \iff \exists c, n_0: T(n) \leq c \cdot f(n) \forall n \geq n_0$
+
+1. By using the change of base formula, $log_5n$ can be represent as $log_5n=\frac{log_2n}{log_25}=log_2n*\frac{1}{log_25}$, and $log_2n$ can be represent as $log_2n=\frac{log_5n}{log_52}=log_5n*\frac{1}{log_52}$
+2. Applying the definition of $O$, let $T(n)\in O(log_5n)$, there exists constants $c>0$, and $n_0\ge 1$, and becuase $\frac{1}{log_25}>0$, therefore $log_5n\leq c*log_2n$ for all $n\geq n_0$, and $log_5n\in c\cdot log_2n$
```

Comment:
I don't see how $log_5n\leq c*log_2n$. And how can $log_5n\in c\cdot log_2n$? $c\cdot log_2n$ is not a set.